Assets/Scripts/MenuButtons.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuButtons : MonoBehaviour {
+
+    int currentScene;
+
+	// Use this for initialization
+	void Start () {
+        currentScene = SceneManager.GetActiveScene().buildIndex;
+	}
+	
+    public void PressStartButton()
```

Comment:
This should probably be past tense if it is meant to be called when the player pressed the start button. It sounds like it presses the start button for you in present tense.